certified-connectors/PDF4me/apiDefinition.swagger.json

```diff
@@ -852,6 +852,116 @@
         }
       }
     },
+    "/v2/FlowV2/ExtractResources": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "Extract text and images from the PDF document. https://dev.pdf4me.com/power-automate/documentation/actions/extract-resources/",
+        "operationId": "ExtractResources_V1",
```

Comment:
@chandraarora operationIds should be in Camel Case. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations). 
Please change to "ExtractResourcesV1".